### PR TITLE
Add a note for user installing api platform with symfony flex

### PR DESCRIPTION
**Pull request by louis-cuny**

### CODE REVIEW

**Comment by alanpoulain** [comment]
There is already a note saying  that just before. I don't think it's a good idea to add this note everywhere.

**Comment by louis-cuny** [comment]
I saw the existing note but I didn't guessed it should apply on the next doc elements. Maybe 
improve the first one would be better ?

**Comment by alanpoulain** [comment]
Yes if you find a good sentence, please do.

**Comment by louis-cuny** [comment]
@alanpoulain I let you check, I am not the best in english

File: core/graphql.md [hunk 1]
```diff
@@ -21,8 +21,7 @@ docker-compose exec php sh -c '
 
 You can now use GraphQL at the endpoint: `https://localhost:8443/graphql`.
 
-*Note:* If you used [Symfony Flex to install API Platform](../distribution/index.md#using-symfony-flex-and-composer-advanced-users),
-the GraphQL endpoint will be: `https://localhost:8443/api/graphql`.
+*Note:* If you used [Symfony Flex to install API Platform](../distribution/index.md#using-symfony-flex-and-composer-advanced-users), URLs will be prefixed with /api by default. The GraphQL endpoint for example will be: `https://localhost:8443/api/graphql`.
```
Comment (alanpoulain) [inline core/graphql.md:24]:
```suggestion
*Note:* If you used [Symfony Flex to install API Platform](../distribution/index.md#using-symfony-flex-and-composer-advanced-users), URLs will be prefixed with `/api` by default. For example, the GraphQL endpoint will be: `https://localhost:8443/api/graphql`.
```

**Review by alanpoulain — APPROVED** [review]

**Comment by alanpoulain** [comment]
Thank you @louis-cuny.

**Pull request merged**